deploy/operator/gather.sh

```diff
@@ -111,6 +111,36 @@ function gather_imageset_data() {
   done
 }
 
+function gather_capi() {
+  CAPI_CRS=(agentmachine agentcluster cluster machine machinedeployment machineset nodepool hostedcluster hostedcontrolplane)
+  capi_dir="${LOGS_DEST}/capi"
+  mkdir -p "${capi_dir}"
+
+  for cr in "${CAPI_CRS[@]}"; do
+    work_dir="${capi_dir}/${cr}"
+    mkdir -p "${work_dir}"
+
+    readarray -t objects < <(oc get ${cr} -A -o json | jq -c '.items[]')
```

Comment:
CAPI related CRDs exists only when the CAPI test was executed
this command `oc get ${cr} -A -o json` might fail

Reply:
See
https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_assisted-service/3989/pull-ci-openshift-assisted-service-master-edge-e2e-metal-assisted-operator-ztp/1538902134928445440